featuregate/controller/pkg/featuregate/featuregate_controller.go

```diff
@@ -1,7 +1,7 @@
 // Copyright 2021 VMware, Inc. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package controllers
+package featuregate
```

Comment:
Ditto: `featuregate/controller/pkg/featuregate` is too repetitive. Suggestions?

Reply:
Maybe `gate`? It was one of @rajathagasthya's import aliases.